src/app/jscaip/GameNode.ts

```diff
@@ -133,6 +135,13 @@ export class GameNode<M extends Move, S extends GameState> {
             this.cache.set(key, value);
         }
     }
+    public getConfig(): C {
+        if (this.config.isPresent()) {
+            return this.config.get();
+        } else {
+            return {} as C;
```

Comment:
maisssss c'pas safe du tout c'machin lo. Pourquoi pas retourner this.config et le parent décide de faire quelque chose si il y a pas de config ?

Reply:
ou pas avoir une config optionelle mais une config vide par défaut ?